%YAML:1.0
---
image_width: 1920
image_height: 1080
board_width: 17
board_height: 13
square_size: 50.
flags: 16385
camera_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 1.0096461088041725e+03, 0., 9.3689248385778296e+02, 0.,
       1.0096581504805509e+03, 4.9988257013426869e+02, 0., 0., 1. ]
distortion_coefficients: !!opencv-matrix
   rows: 8
   cols: 1
   dt: d
   data: [ 9.5856164145301237e-01, 1.8914835820355605e-01,
       -2.4506997072279488e-06, 5.9923693345781568e-05,
       3.9186197392106761e-03, 1.3529369410484064e+00,
       4.6988473344466519e-01, 3.3337587992892999e-02]
